README.md

```diff
@@ -19,10 +19,10 @@ the Linux Kernel Virtual Machine (KVM) to create and run microVMs. Firecracker
 has a minimalist design. It excludes unnecessary devices and guest-facing
 functionality to reduce the memory footprint and attack surface area of each
 microVM. This improves security, decreases the startup time, and increases
-hardware utilization. Firecracker currently supports Intel CPUs, with planned
-AMD and Arm support. Firecracker will also be integrated with popular container
-runtimes.
-
+hardware utilization. Firecracker currently supports Intel, AMD and Arm CPUs. 
+Firecracker can also be integrated in container runtimes. Checkout 
```

Comment:
nit:
```suggestion
Firecracker can also be integrated in container runtimes. Check out 
```

Reply:
Fixed it here: https://github.com/firecracker-microvm/firecracker/compare/97edf46d827772c879f2d0a00c67df2597279cd4..909692079f3b6d3fc894f9e74ffbc0bf023403c2.